modules/core/shared/src/main/scala/lucuma/core/geom/jts/JtsShapeInterpreter.scala

```diff
@@ -20,7 +20,7 @@ import java.util.Collections
 object JtsShapeInterpreter extends ShapeInterpreter {
 
   private val EmptyGeometry: Geometry =
-    Jts.geometryFactory.buildGeometry(Collections.emptyList[Geometry])
+    Jts.geometryFactory.createPoint
```

Comment:
This change is needed or many of the tests break

Reply:
I don't really understand what an empty point means but I suppose if it works ...

Reply:
They are in fact different entites a point is just a point:
```
println(Jts.geometryFactory.createPoint)
   POINT EMPTY```

While the previous one is an empty geometry collection
```
println(Jts.geometryFactory.buildGeometry(Collections.emptyList[Geometry]))
   GEOMETRYCOLLECTION EMPTY
```

I'm not sure the reason why but the updated JTS lib cannot do operations like union or intersection on GEOMETRYCOLLECTION